Updated explainer (#1216)
ethereum-optimism/docs: pages/stack/interop/architecture.mdx

```

OP-Supervisor holds a database of all the log events of all the chains in the interoperability cluster.

> Every event can potentially initiate a cross-domain message, and it is the job of OP-Supervisor to validate that the log event really happened on the source chain.

Reviewer: ```suggestion
Every event can potentially initiate a cross-domain message, and it's the job of OP-Supervisor to validate that the log event really happened on the source chain.
```

Author: Thank you for doing the contractions. Sorry I haven't done them, but I've been sick.
